ext/curl/tests/curl_version_basic_001.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+Test curl_version() basic functionality
+--CREDITS--
+Rodrigo Prado de Jesus <royopa [at] gmail [dot] com>
+#PHPSP User Group Brazil
```

Comment:
For the future: We don't really use `--CREDITS--` anymore, because this information is already in git, in a better form.

Reply:
Ok, removed the credits area @nikic